torch/csrc/jit/codegen/cuda/compute_at_map.h

```diff
@@ -101,9 +124,17 @@ class TORCH_CUDA_CU_API ComputeAtMap {
   std::unordered_map<std::shared_ptr<std::deque<IterDomain*>>, ParallelType>
       parallel_type_map_;
 
-  // For each IterDomain set we will track how many concrete root domains were
-  // used to generate the IterDomain
+  // One iteration domain with the largest count in concrete_id_count_map_
+  // within a disjoint set will be selected as the "concrete_id" of that set
   std::unordered_map<IterDomain*, IterDomain*> concrete_id_map_;
+
+  // Track how many concrete IDs compose each iteration domain. If in the
+  // rfactor domain of a view operation this value must be maximum 1 (can be 0
+  // if it's a broadcast). concrete_id_count_map_ is only generated for an index
+  // map. In order to select concrete_id_map_ a compute at map with index mode
+  // must be present. The Index map can come in through a GPU Lower object,
+  // otherwise it will be generated if using Parallel or Loop mapping modes.
+  std::unordered_map<IterDomain*, std::pair<int, int>> concrete_id_count_map_;
```

Comment:
Is `concrete_id_count_map_` only created for an index map? It seems it's always created.

Reply:
Stale, will remove.